tests/bears/BearTest.py

```diff
@@ -396,16 +398,18 @@ def test_download_cached_file(self):
         filename = self.filename
         file_location = self.file_location
 
-        with requests_mock.Mocker() as reqmock:
+        with freeze_time('2017-01-01 00:00:00.00') as frozen_datetime, \
```

Comment:
It's not. There was a `sleep` before. This will mock system's date. You can understand it better here: https://github.com/spulec/freezegun